docs/Making_presets.md

```diff
@@ -59,3 +59,13 @@ The package name should should follow this pattern: `mrm-preset-<TASK>`, otherwi
 mrm license --preset unicorn # mrm-preset-unicorn
 mrm license --preset @mycompany/unicorn-preset # @mycompany/unicorn-preset
 ```
+
+## Default preset
+
+The default preset is `mrm-preset-default`, you can set a custom default preset in `.mrm/config.json`:
```

Comment:
This is no longer true, `mrm-preset-default` is deprecated and tasks are autoinstalled by default.

Reply:
I've updated the documentation, let me know if you want additional changes